src/core/1/plugins/plugin-context/constructors/request/index.md

```diff
@@ -20,7 +20,7 @@ For more information about this object, refer to its [technical documentation](h
 
 Network protocol specific headers can be added to the response. If the protocol supports it, these headers are forwarded in the response sent to the client.
 
-As Kuzzle supports the HTTP protocol natively, the Request object handles HTTP headers special cases.  
+As Kuzzle supports the HTTP protocol natively, the Request object handles HTTP headers special cases.
```

Comment:
Has this change been automatically made by your editor?

Reply:
Yes, and it's bad for markdown files!

Reply:
```
    "files.trimTrailingWhitespace": true,
    "[markdown]": {
        "files.trimTrailingWhitespace": false
    },
```
To vscode config

Reply:
I made the same change a few months ago in sublime :grin: